project/project.go

```diff
@@ -136,8 +136,8 @@ func (p *Project) Load(bytes []byte) error {
 	return nil
 }
 
-func (p *Project) loadWrappers(wrappers map[string]*serviceWrapper) error {
-	for _, name := range p.reload {
+func (p *Project) loadWrappers(wrappers map[string]*serviceWrapper, list []string) error {
+	for _, name := range list {
```

Comment:
will do.  It is the list of service names to initialize.  FYI, I know `project.go` is highly unreadable.  I plan to rewrite it.